docs/manual/docs/sample-scripts/spo/add-custom-clientside-webpart-to-modern-page.md

```diff
@@ -0,0 +1,53 @@
+# Add custom client-side WebPart to modern page
+
+Author: [Yannick Plenevaux](https://ypcode.wordpress.com)
+
+You've built an amazing new WebPart and now you want to programatically add it to a modern page. 
```

Comment:
Typo in `programmatically`

Reply:
Copy pasted the description from the original discussion, completely missed the typo, thank you for fixing it :)